marimo/_server/export/__init__.py

```diff
@@ -77,7 +77,7 @@ def export_as_script(path: MarimoPath) -> ExportResult:
 def export_as_md(path: MarimoPath) -> ExportResult:
     ir = _as_ir(path)
```

Comment:
should `_as_ir` be putting the filename in the `NotebookSerializationV1` object, instead of passing it down again to `to_markdown`